firmware/controllers/algo/engine_type_impl.cpp

```diff
@@ -413,6 +413,22 @@ void applyEngineType(engine_type_e engineType) {
 	default:
 	  applyEngineTypeExt(engineType);
 	}
+
+#if defined(HW_NUCLEO_F767) || defined(HW_NUCLEO_H743)
```

Comment:
no-no-no, this can go to many places but not here

Reply:
> no-no-no, this can go to many places but not here
Can I create a new engine type and put the initialization there? Or is it better with a console command? I need something similar to Jest.js `beforeEach`
I have to run that every time an engine type is set in the tests and before the test is executed